_posts/0000-01-02-Glaciemdeus.md.

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Can't find the button to edit the file? It may look like a pencil, or it may look like three dots.